src/client/datascience/notebook/kernelProvider.ts

```diff
@@ -135,7 +137,7 @@ export class VSCodeKernelPickerProvider implements NotebookKernelProvider {
                 undefined,
                 token
             ),
-            this.interpreterService.getActiveInterpreter(document.uri)
+            isPythonNb ? this.interpreterService.getActiveInterpreter(document.uri) : Promise.resolve(undefined)
```

Comment:
Won't this still ask if the notebook is a python notebook? What we really want is to do nothing with the interpreters unless the python extension is installed. See IPythonExtensionChecker

Reply:
Oh ok, thanks didn't expect that.
Makes sense

Reply:
This doesn't seem to be resolved yet.

Reply:
Damn it, yes

Reply:
Done